pliers/tests/extractors/test_audio_extractors.py

```diff
@@ -352,3 +357,60 @@ def test_percussion_extractor():
     assert np.isclose(df['onset'][29], 1.346757)
     assert np.isclose(df['duration'][29], 0.04644)
     assert np.isclose(df['percussive'][29], 0.004497, rtol=1e-4)
+
+
+@pytest.mark.parametrize('hop_size', [0.1, 1])
+@pytest.mark.parametrize('top_n', [5, 10])
+@pytest.mark.parametrize('target_sr', [22000, 14000])
+def test_audioset_extractor(hop_size, top_n, target_sr, yamnet_path=None):
```

Comment:
Does this currently test for alternate values of `yamnet_path`? If not, that seems worth adding.

Reply:
Having removed the yament_path argument, I guess this is now outdated.